### PR TITLE
feat: Make Asset Search Less Fuzzy and Remove Skeleton for Swap Details 

### PR DESCRIPTION
**Pull request by JoseRFelix**
## What is the purpose of the change

image

### ClickUp Task

[ClickUp Task URL](https://app.clickup.com/t/86a1g2bb3)

### Testing

- [ ] Totally unrelated assets should not be displayed when searching on the swap tool. For E.g. 'Alter' showing up when searching for 'ether'

### CODE REVIEW

File: packages/web/components/drawers/token-select-drawer.tsx [hunk 1]
```diff
@@ -83,11 +83,6 @@ export const TokenSelectDrawer: FunctionComponent<{
       setAssets(swapState.selectableAssets);
     }, [isRequestingClose, swapState.selectableAssets]);
 
-    // maintain focus on search box while typing across re-renders as assets update
```
Comment (MaxMillington) [inline packages/web/components/drawers/token-select-drawer.tsx:86]: Was this not working?

File: packages/web/components/swap-tool/index.tsx [hunk 2]
```diff
@@ -450,6 +451,7 @@ export const SwapTool: FunctionComponent<SwapToolProps> = observer(
                       (tokenDenom: string) => {
                         swapState.setFromAssetDenom(tokenDenom);
                         closeTokenSelectDropdowns();
+                        fromAmountInputEl.current?.focus();
```
Comment (MaxMillington) [inline packages/web/components/swap-tool/index.tsx:454]: Ah I see.

File: packages/web/hooks/data/data-filter.ts [hunk 3]
```diff
@@ -13,6 +13,7 @@ export class DataFilter<TData> implements DataProcessor<TData[]> {
       keys: keys,
       findAllMatches: true,
       useExtendedSearch: true,
+      threshold: 0.2,
```
Comment (MaxMillington) [inline packages/web/hooks/data/data-filter.ts:16]: maybe leave a comment here

File: packages/web/server/queries/complex/assets/index.ts [hunk 4]
```diff
@@ -74,6 +74,7 @@ export async function getAssets({
       if (params.search) {
         const fuse = new Fuse(assets, {
           keys: searchableKeys,
+          threshold: 0.2,
```
Comment (MaxMillington) [inline packages/web/server/queries/complex/assets/index.ts:77]: same

**Review by MaxMillington — APPROVED** [review]

**Review by arispech — APPROVED** [review]
LGTM

Screenshot 2023-12-11 at 10 45 15 AM
Screenshot 2023-12-11 at 10 47 08 AM

**Review by CryptoAssassin1 — APPROVED** [review]
looking good.
Screenshot from 2023-12-11 20-57-57
Screenshot from 2023-12-11 20-56-51

File: packages/web/components/swap-tool/index.tsx [hunk 5]
```diff
@@ -106,10 +106,14 @@ export const SwapTool: FunctionComponent<SwapToolProps> = observer(
     const routesVisDisclosure = useDisclosure();
 
     const [showQuoteDetails, setShowEstimateDetails] = useState(false);
-    /** User has input and there is enough liqudity and routes for given input. */
-    const isQuoteDetailRelevant =
+
+    const isInputAmountGreaterThanZero =
```
Comment (jonator) [inline packages/web/components/swap-tool/index.tsx:110]: FYI `inAmountInput` has an `isEmpty` flag that may be useful. Not sure though.

**Review by jonator — APPROVED** [review]
NIT

**Review by kamal-sutra — APPROVED** [review]

File: packages/web/hooks/data/data-filter.ts (hunk 3 above)
Comment (JoseRFelix) [reply packages/web/hooks/data/data-filter.ts:17]: Thanks for the feedback! I added the comment on my last commit

File: packages/web/components/swap-tool/index.tsx (hunk 5 above)
Comment (JoseRFelix) [reply packages/web/components/swap-tool/index.tsx]: I'll just revert it to the changes in prod.

**Pull request merged**